packages/eds-core-react/src/components/Checkbox/Checkbox.test.tsx

```diff
@@ -38,6 +39,10 @@ describe('Checkbox', () => {
     const { asFragment } = render(<Checkbox label={'checkbox'} />)
     expect(asFragment()).toMatchSnapshot()
   })
+  it('should pass a11y test', async () => {
```

Comment:
Could we also get one a11y test with external `label`, its been important for us to support this use-case as its used for forms and tables etc.